system/jhtree/jhtree.cpp

```diff
@@ -112,7 +112,7 @@ SegMonitorList::SegMonitorList(const SegMonitorList &from, const char *fixedVals
 
 void SegMonitorList::describe(StringBuffer &out) const
 {
-    for (unsigned idx=0; idx < lastRealSeg(); idx++)
+    for (unsigned idx=0; idx <= lastRealSeg(); idx++)
```

Comment:
Won't this crash if there are no segment monitors? (I think lastRealSeg still returns 0 and the item call will fail.)

Reply:
Not sure it is ever called in a case where no filters, but I will look at making more robust